### PR TITLE
add columns, logo, modify scrollbar

### PR DESCRIPTION
**Pull request by psgb-r3763**
This PR does the following:

- Adds column functionality. Usage is explained on the [notion columns tutorial page.](https://templecodex.notion.site/Using-Columns-in-Ghost-WIP-ad3173c0e06c465bbecdbd66c251ddf6)
- Makes the scroll bar darker
- Adds the Temple Codex logo to the bottom global footer
  - Image is easily replaced: `default.hbs:69` refers to `codexlogo.png` in  `/assets/`

Example of columns:

Example of Temple Codex logo at the bottom of a post:

### Future Plans:
In the future, I'm thinking we could widen all of the columns a bit more. As long as the valid combinations:

1. don't add up to more than 100%, 
2. have enough white space (maybe the valid combinations should add up to around 96%? I believe this was the standard for the simple-grid library initially)

it should be fine.

### CODE REVIEW

**Comment by garlicbreadketchup** [comment]
Currently re-writing columns using CSS multi columns; putting PR on hold for now

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 node_modules/
+assets/built/
+.gitignore
```
Comment (zb-bebop) [inline .gitignore]: @garlicbreadketchup Thanks for adding `assets/built` to this! Good idea.

Looks like you've also added `.gitignore` to itself though - I'd think we want to strike line 3? We may need to expand the `.gitignore` down the road.

Comment (zb-bebop) [inline .gitignore]:
```suggestion
```

Comment (psgb-r3763) [reply .gitignore]: Absolutely agreed!

**Review by zb-bebop — APPROVED** [review]

**Pull request merged**